### ISSUE TITLE
Feature documentation

### ISSUE THREAD

**Issue by JoeMurray** [veda-consulting/uk.co.vedaconsulting.outlookapi#37]
Could you update the README.md to include the full current feature set? Skimming one of the code files I see that in addition to replicating the email to activity functionality of CiviCRM core it seems to also support:
- something with Cases
- something with reading activities in CiviCRM
- synchronizing contacts between Outlook Groups and CiviCRM Groups that are somehow configured to be synched with Outlook.

I don't see the VBA code in the repo, just a zip file, which is part of why I'm not sure what the extension does inside Outlook. Is there a public source code repo for it so we could examine what the code running in Outlook does? Also, what license is that code released under and are you interested in pull requests for fixes or feature enhancements?

**Comment by veda-consulting** [comment]
@JoeMurray I agree, we're adding some new features and will aim to get that all documented into docs.civicrm.org. We'll also include a public repo for the code.
